zulip_bots/zulip_bots/lib.py

```diff
@@ -156,7 +158,58 @@ def run_message_handler_for_bot(lib_module, quiet, config_file, bot_name):
 
     state_handler = StateHandler()
 
+    # Bot details and default commands from defaults, then override if provided
+    bot_details = { 'name': bot_name.capitalize(),
+                    'description': "",
+                    'commands': {},
+                    'no_defaults': False,
+    }
+    def def_about():
+        desc = bot_details['description']
+        return "**{}**{}".format(bot_details['name'],
+                                 "" if desc == "" else ": {}".format(desc))
```

Comment:
Interesting construction; I just played with this but it doesn't seem to work in practice as "not desc" gives True :)

Alternatively, just:
```
        if bot_details['description'] == "":
            return "**{}**".format(bot_details['name'])
        return "**{}**: {}".format(bot_details['name'], bot_details['description'])
```
?